src/tasks/tasks.py

```diff
@@ -455,6 +455,13 @@ def update_metrics(self, logits, labels, tagmask=None):
         self.scorer2(logits, labels)
         return
 
+# register cola-like tasks with a non-decorator syntax
+# put data for colax in a subfoler named colax under CoLA 
+from .__init__ import ALL_COLA_NPI_TASKS
+for cola_npi_x in ALL_COLA_NPI_TASKS:
+    CoLATask = register_task(cola_npi_x, rel_path='CoLA/%s/' % cola_npi_x)(CoLATask)
```

Comment:
Right now this only creates one task. In any case, though, why not just name all of them? We can commit changes if we change the data—that's how it works for every other task here.

Reply:
there will be more --- the idea is to put all hand-coding into `__init__.py`, which we need to do anyway if we want these added tasks to be taken care by `write_preds()` in `evaluate.py`

Reply:
Let's just treat this the same way as MNLI, even if that means naming things in multiple places: Register everything where we define the task. Handling different tasks different ways will make this much harder to maintain later.

Also, are you sure the set isn't final? I think we need to set the code up for what the NPI team has now either way.

Reply:
Updated!

Reply:
I mean the train-dev-test split has not been out for each environment yet

Reply:
Hrm. Bug the data folks? It was supposed to be done by this point... (@alexwarstadt — do you know where this stands?)

Reply:
Split data is now on the data_generation repo: https://github.com/alexwarstadt/data_generation/tree/master/outputs/npi/environments/splits, with thanks to @sheng-fu